src/main/resources/diagram_git_starship.puml

```diff
@@ -2,132 +2,162 @@
 
 package rape.brutal.gitstarship {
 
-    package ships {
-
-        Starship <|-- GitStarship
-        Starship <|-- PirateStarship
-
-        abstract class Starship {
-            #Engine engine;
-            #Hull hull;
-            #Storage<IStoreable> storage;
-
-            +boolean putInStorage(IStoreable iStoreable);
-            +IStoreable getFromStorage(String iStoreableName);
-
-            -void flyTo(IReachable iReachable);
+    class Universe {
+        -int kills;
+        -int spends;
+        +void startUniverse();
+        +void win();
+        +void lose();
```

Comment:
It is not fixed in last commit (49337ec9bd6e7fe8582bf00650133d44e242ae2b). @Alvahaze